balancer-labs/frontend-v2
# Feature/pk vebal header

# Description

Includes a new header for the veBAL page. 

## Type of change

- [ x ] Other: UI improvements

## How should this be tested?

Go to the`/vebal` page and make sure things look good at different screen sizes.
